src/Sylius/Bundle/ApiBundle/Resources/config/services/context_builders.xml

```diff
@@ -78,7 +78,8 @@
         </service>
 
         <service
-            id="Sylius\Bundle\ApiBundle\SerializerContextBuilder\LoggedInShopUserIdAwareContextBuilder"
+            id="sylius_api.serializer_context_builder.logged_in_shop_user_id_aware"
+            class="Sylius\Bundle\ApiBundle\SerializerContextBuilder\LoggedInShopUserIdAwareContextBuilder"
```

Comment:
Similar as above, it would be nice to move all context builders to:
```suggestion
            class="Sylius\Bundle\ApiBundle\Serializer\ContextBuilder\LoggedInShopUserIdAwareContextBuilder"
```